mastracode/web/ui/styles.css

```
@@ -0,0 +1,2034 @@
+  border: 1px solid var(--border);
+  border-radius: var(--radius-sm);
+  color: var(--fg);
+  font-size: 12.5px;
+  font-weight: 600;
+  padding: 6px 10px;
+  cursor: pointer;
+}
+.settings-select:focus-visible { outline: 2px solid var(--accent); outline-offset: 1px; }
+
+/* Allow the panel to grow with the providers list, but cap + scroll. */
+.settings-panel { display: flex; flex-direction: column; max-height: 86vh; }
```

Stylelint (17.13.0)

[error] Too many declarations, maximum 1 (declaration-block-single-line-max-declarations)

(declaration-block-single-line-max-declarations)